tutorials/app.html

```diff
@@ -40,13 +40,19 @@
     <strong>Tutorial:</strong> Using the ROB 102 Navigation Web App
   </header>
 
+<<<<<<< HEAD
+  <p>
+    Coming sooner! 
+  </p>
+=======
```

Comment:
This is left over from the merge conflict. Can you remove lines 43 to 47 and line 55 to resolve the conflict?